### PR TITLE
Doc for apps types

### PR DESCRIPTION
**Pull request by Exelord**
# Description
The basic explanation of apps types.
As from now, the only difference will be in apps displaying.

### CODE REVIEW

**Comment by Exelord** [comment]
@Azdaroth Should be merged now, I think so

File: content/reference/requirements.md [hunk 1]
```diff
@@ -12,11 +12,21 @@ If you are not a partner yet, it's [free to signup so join us :)](https://www.bo
...
 **Note**: BookingSync traditional Users have to input `private access code` in their Apps section to start using your private application. That code is automatically generated for you and is visible in your applications manage section.
 
+## Applications types
+
+### Embedded
+Embedded applications are rendered within an iframe on BookingSync.com.
+**You application needs to allow on embedding in `iframe`.**
+
+### Standalone
+Standalone applications will be working outside of BookingSync website. It means that they will be opened in a new window.
```
Comment (Azdaroth) [inline content/reference/requirements.md]: Not sure if this is the best way to describe it, how about "it means that they will be opened outside the BookingSync Admin UI"?

Comment (Exelord) [reply content/reference/requirements.md]: They don't know at the moment what Admin UI means exactly.

Comment (Azdaroth) [reply content/reference/requirements.md]: @Exelord Sounds good to me 👍

**Comment by Exelord** [comment]
merge? :)

**Comment by Azdaroth** [comment]
@Exelord Lol, right :P

Can you please handle the release? A changelog update would be useful as well.

**Comment by Exelord** [comment]
Sure. WIth a pleasure!

**Pull request merged**